bin/pjv

```diff
@@ -7,46 +7,34 @@
  */
 
 // Command line options
-var args = require("optimist")
+var options = require("yargs/yargs")(process.argv.slice(2))
 	.options("filename", {
 		default: "package.json",
 		alias: "f",
+		description: "package.json file to validate",
 	})
-	.describe("filename", "package.json file to validate")
 	.options("spec", {
 		default: "npm",
 		alias: "s",
+		description: "spec to use - npm|commonjs_1.0|commonjs_1.1",
 	})
-	.describe("spec", "which spec to use - npm|commonjs_1.0|commonjs_1.1")
 	.options("warnings", {
 		default: false,
 		alias: "w",
+		description: "display warnings",
 	})
-	.describe("warnings", "display warnings")
 	.options("recommendations", {
 		default: false,
 		alias: "r",
+		description: "display recommendations",
 	})
-	.describe("recommendations", "display recommendations")
 	.options("quiet", {
 		default: false,
 		alias: "q",
+		description: "less output",
 	})
-	.describe("quiet", "less output")
 	.usage("Validate package.json files")
-	.options("help", {
-		default: false,
-		alias: "h",
-	})
-	.describe("help", "this help message")
-	.alias("help", "?");
-
-var options = args.argv;
-
-if (options.help) {
-	args.showHelp();
-	process.exit(0);
-}
+	.parse();
```

Comment:
🤔 Am I doing something wrong? When I put in a `console.log({ options })` here, `recommendations` and `warnings` aren't detected:

```plaintext
joshgoldberg ~/repos/package.json-validator $ node bin/pjv -r -w   
{
  options: {
    _: [],
    r: false,
    recommendations: false,
    w: false,
    warnings: false,
    filename: 'package.json',
    f: 'package.json',
    spec: 'npm',
    s: 'npm',
    quiet: false,
    q: false,
    '$0': 'bin/pjv'
  }
}
{ valid: true }
joshgoldberg ~/repos/package.json-validator $ node bin/pjv --recommendations --warnings
{
  options: {
    _: [],
    recommendations: false,
    r: false,
    warnings: false,
    w: false,
    filename: 'package.json',
    f: 'package.json',
    spec: 'npm',
    s: 'npm',
    quiet: false,
    q: false,
    '$0': 'bin/pjv'
  }
}
```

They are on `main`.